dpcpp/base/config.hpp

```diff
@@ -65,6 +65,11 @@ struct config {
      * to maximize GPU occupancy.
      */
     static constexpr uint32 min_warps_per_block = 4;
+
+    /**
+     * The maximal number of threads allowed in a CUDA warp.
```

Comment:
DPCPP is also 256 ?

Reply:
no, it depends on the device. but 256 is common acceptable block size for dpcpp device.
it is only for pushing dpcpp in the first step.

Reply:
Okay, that makes sense. Maybe you can update the doc strings ?